Enderal + Enderal SE Support

jake770321 opened the pull request:
Didn't work on my existing install but after a clean install everything seems to work perfectly. Installed 3 different mods and made sure they were working in game. First time doing something this technical so please check everything if possible.

I don't think the last two lines in enderal.sh are needed as the game comes with the script extender preinstalled but removing these lines or setting them to blank breaks the installer. Not a huge deal though as everything still works.

Was also able to get the launcher working by following the steps here "https://github.com/rockerbacon/modorganizer2-linux-installer/blob/master/post-install.md" and adding an underscore. ENBoost is easily disabled from this launcher with a single click. 

If everything here looks alright then I'll go ahead and do the same for the special edition.
Thank you so much for making this installer and making the process of adding games so easy!

rockerbacon reviewed (changes requested):
I'll see if I can make a small patch to remove the need for downloading the script extender.

jake770321 commented on step/select_game.sh:
```diff
@@ -24,6 +24,7 @@ selected_game=$( \
 		"oblivion" "Oblivion" \
 		"skyrim" "Skyrim" \
 		"skyrimspecialedition" "Skyrim Special Edition" \
+		"enderal" "Enderal: Forgotton Stories" \
```
Accidentally requested review before making changes, sorry. Should be all good now

jake770321 commented:
I've made the change you requested and fixed the typo.

I've also added Enderal SE and made sure everything works. Mods are being installed correctly and the protontricks xaudio tweak is also working (and required as this fixes the NPC voices in the game)

I did have some trouble with it thinking the game was installed on my primary drive when it was in fact installed on an external drive and this caused the installer to not work. Not entirely sure what changed but it eventually started working, I'll try it a few more times and see if I can replicate it. **(This only happened on the Special Edition)**

rockerbacon commented:
Thanks a lot for your contribution!
